### ISSUE TITLE
New release 1.1.3

### ISSUE THREAD

**Issue by Valdnet** [ManeraKai/simplytranslate_mobile#158]
Tasks to be performed for version 1.1.3:

- [x] Application tests
- [ ] Application translations
- [ ] [Milestone](https://github.com/ManeraKai/simplytranslate_mobile/milestone/4)

@ManeraKai I think it's time to release a new version with fixed bugs and additional features 😉.

**Comment by Valdnet** [comment]
@ManeraKai I think you can update the translation from POEditor if there is a new one and release a new version. Issue #61 has been shifted for v1.1.4 for an obvious reason 😉.

**Comment by ManeraKai** [comment]
I did update POEditor a few days ago, I should check it again though. I'll release a new version once I finish my exams. 3 left for 3 days.

**Comment by ManeraKai** [comment]
Did a scan to clean the code. Updated translations. Fixed bugs. Almost ready. Any bug on your side @Valdnet?

**Comment by ManeraKai** [comment]
https://github.com/ManeraKai/simplytranslate_mobile/actions/runs/1456830351

**Comment by Valdnet** [comment]
The icon for a light theme should be of a different color. Currently confusing the user a bit.

**Comment by ManeraKai** [comment]
Any suggestion?

**Comment by Valdnet** [comment]
Maybe yellow?

**Comment by ManeraKai** [comment]
Yellow doesn't contrast at all with white, I think I tried that, works well with dark mode though.

**Comment by ManeraKai** [comment]
A dimmed yellow maybe?
<img width=360 src="https://user-images.githubusercontent.com/40805353/141659018-11993390-f9d8-41dc-b5af-c9ec57d73467.png">

**Comment by ManeraKai** [comment]
https://github.com/ManeraKai/simplytranslate_mobile/actions/runs/1457412702

**Comment by Valdnet** [comment]
You need to change the font color in the theme picker in dark theme.

**Comment by ManeraKai** [comment]
Changed: https://github.com/ManeraKai/simplytranslate_mobile/actions/runs/1458727811

**Comment by Valdnet** [comment]
Error bolid.

**Comment by ManeraKai** [comment]
Idk what's wrong, reran it: https://github.com/ManeraKai/simplytranslate_mobile/actions/runs/1458880402

**Comment by Valdnet** [comment]
There are 4 strings in POEditor that have no text to translate:

![obraz](https://user-images.githubusercontent.com/47037905/141682151-cea282e5-cb41-4639-bbdd-7ffd368de801.png)

**Comment by ManeraKai** [comment]
They are not in the `.arb` files, I'll fix them but they don't have an effect on our release I think.

**Comment by Valdnet** [comment]
Change the font color from black to white in the theme selector for the dark theme:

![obraz](https://user-images.githubusercontent.com/47037905/141682363-60e88433-6385-43c2-ac06-4c2c683d88aa.png)

**Comment by ManeraKai** [comment]
Fixed.

**Comment by Valdnet** [comment]
These text strings should be added to the translation:

https://github.com/ManeraKai/simplytranslate_mobile/blob/8c08cf015dbf166366d277edbcb8dbaf05a121af/lib/screens/about/about_screen.dart#L89

https://github.com/ManeraKai/simplytranslate_mobile/blob/8c08cf015dbf166366d277edbcb8dbaf05a121af/lib/screens/about/about_screen.dart#L117

**Comment by ManeraKai** [comment]
There brand names. i.e: `Buy me a coffee` has it's domain name as https://www.buymeacoffee.com/

**Comment by Valdnet** [comment]
Can there be Arabic letters or a text string as RTL in other languages, such as Arabic?

**Comment by ManeraKai** [comment]
Your right, I checked https://www.buymeacoffee.com/, didn't have an Arabic name. Checked https://liberapay.com/, it has an Arabic name `ليبرا باي
`

**Comment by Valdnet** [comment]
So we give the translation 😁.

**Comment by Valdnet** [comment]
I would also add: `Buy me a coffee`. It is not known how it occurs in other languages 😉.

**Comment by ManeraKai** [comment]
It's untranslated now in their website. If it gets translated, I'll added it 👍

**Comment by Valdnet** [comment]
Still need to change to `about_screen.dart`.

**Comment by Valdnet** [comment]
Still need to change to `about_screen.dart`.

**Comment by ManeraKai** [comment]
oh I forgot.

**Comment by ManeraKai** [comment]
Fixed.

**Comment by Valdnet** [comment]
The application is named as `SimplyTranslate Mobile` and the name in the application bar is `Simply Translate Mobile`:

![obraz](https://user-images.githubusercontent.com/47037905/141693064-ab80bf8a-18bb-4bb6-8a9b-ad671b839982.png)

 Is that correct?

**Comment by ManeraKai** [comment]
@Valdnet I'll release the version. Hope you're still alive.

**Comment by Valdnet** [comment]
@ManeraKai Of course, release the new version :+1:.

> Hope you're still alive.

I am 😉.

**Comment by Aga-C** [comment]
Well done, guys 🙂 
BTW. The website is really nice too 👍

**Comment by Valdnet** [comment]
Yesterday a new version of the application was released 😁.

> Well done, guys

Thanks 😉.

**Comment by ManeraKai** [comment]
And polish: https://f-droid.org/pl/packages/com.simplytranslate_mobile/

**Comment by ManeraKai** [comment]
Talking about the web version of the f-droid site, it's totally ruined in the f-droid app.
context:
```
|-------------------------|
| Talking about this |
|-------------------------|
```

**Comment by Valdnet** [comment]
It doesn't look good. I'd rather give it up.

**Comment by ManeraKai** [comment]
Sadly

**Comment by Valdnet** [comment]
On the application page, you need to update the screenshots. 😉.

**Comment by ManeraKai** [comment]
Yeah I noticed that. Updated them 3 days ago: https://github.com/ManeraKai/simplytranslate_mobile/commit/91a0f9ee5edb1c7a4a0943aea81c80d1367f7b10

**Comment by Valdnet** [comment]
I'm talking about the screenshots on the website, not Github.

**Comment by ManeraKai** [comment]
Updated them: https://github.com/ManeraKai/simplytranslate_mobile/commit/55124ae2847fd72e7f2fcf83388dc821c8503b4d
